src/pyvesync/vesync.py

```diff
@@ -377,12 +396,12 @@ def energy_today(self):
     @abstractmethod
     def power(self):
         """Return current power in watts"""
-        return self.details.get('power')
+        return float(self.details.get('power'))
```

Comment:
self.details.get('power') may return None if I recall correctly so this would lead to a error in that case

Reply:
should I return 0 in case of error?